stdlib/asyncio/events.pyi

```diff
@@ -211,7 +211,14 @@ class AbstractEventLoop:
     # Network I/O methods returning Futures.
     @abstractmethod
     async def getaddrinfo(
-        self, host: str | None, port: str | int | None, *, family: int = ..., type: int = ..., proto: int = ..., flags: int = ...
+        self,
+        host: bytes | str | None,
```

Comment:
Any reason not to include `bytearray` in the union as well, like we do in `socket.getaddrinfo()`?

```suggestion
        host: bytearray | bytes | str | None,
```

Reply:
`bytearray` is [redundant with `bytes` in argument types](https://docs.python.org/3/library/typing.html#typing.ByteString) and shouldn't be included.

Reply:
> `bytearray` is [redundant with `bytes` in argument types](https://docs.python.org/3/library/typing.html#typing.ByteString) and shouldn't be included.

Okay — no action to be taken here then; we should just change `socket.getaddrinfo`